packaging/smp/config/server.xml.template

```diff
@@ -82,9 +82,9 @@
                protocol="org.apache.coyote.http11.Http11NioProtocol"
                port="{{tomcat_port}}" maxThreads="200"
                scheme="https" secure="true"
-               keystoreFile="/etc/harmony-smp/tls-keystore.jks"
+               keystoreFile="/etc/harmony-smp/tls-keystore.p12"
```

Comment:
In the "context.xml" the value for "smp.keystore.type" is inserted to the db configuration table as "JKS", but here it is "P12"?